public/icons/name.d.ts

```diff
@@ -71,6 +71,7 @@
 	| "integration/full"
 	| "integration/partial"
 	| "key"
+	| "lightning_sidebar"
```

Comment:
```suggestion
	| "lightning_navbar"
```

This will be better, since soon we will have a horizontal nav bar as an option of the page layout.